src/components/CheckoutPayment/formOptions.js

```diff
@@ -0,0 +1,50 @@
+// Generates options for mont and year expiry selects
```

Comment:
I didn't understand why you removed the other file with the functions to generate the years and months, made a JsPerf to compare and these new ones are 42% slower

[JsPerf comparison](https://jsperf.com/buildmonthsyears/1)

Reply:
Code is more readable that way and I don't have to mock anything for tests since all functions are pure. All functions run once per component mount and their execution time is like 0.0000001 ms so not sure if 'performance' argument is valid here.

Reply:
fair enough 👍